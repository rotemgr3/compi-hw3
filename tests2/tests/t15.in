//piazza 84 - should be errorDef  +
int foo(int x, int y) {int a = 5;}
void foo(int z, int x) {int a = 5;}
void main() {int a = 5;}

