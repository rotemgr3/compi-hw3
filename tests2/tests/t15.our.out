---end scope---
x INT -1
y INT -2
a INT 0
line 3: identifier foo is already defined
